lib/cache-utils.js

```diff
@@ -9,20 +9,16 @@ export const State = {
   CacheMatchedKey: "matched_key",
 };
 
-export function getCacheDirectory() {
-  const platform = os.platform();
-  const home = process.env.HOME;
-
-  switch (platform) {
-    case "linux":
-      return path.join(home, ".cache/scarb");
-    case "darwin":
-      return path.join(home, `Library/Caches/com.swmansion.scarb`);
-    case "win32":
-      return path.join(process.env.APPDATA, "swmansion/scarb/config");
-    default:
-      throw new Error(`caching is not available on this platform: ${platform}`);
+export async function getCacheDirectory() {
```

Comment:
Can you please add `await` before `getCacheDirectory` also here? [`await cache.saveCache([getCacheDirectory()], primaryKey);`](https://github.com/ametel01/setup-scarb/blob/2ba10e70db8028e48ca2f656dae04d2a782c4b73/lib/cache-save.js#L12) I believe it's needed, and currently we're caching by promise object address, not its computed value per se.

Reply:
done